### ISSUE TITLE
[Bug] Crt_sh_Transparency_Logs_1_0 - No JSON object could be decoded

### ISSUE THREAD

**Issue by KRUXLEX** [TheHive-Project/Cortex-Analyzers#594]
**Describe the bug**
For few weeks I start experience issue with this analyzer. It works in December, and 3 days ago I use it again and find a problem. It's look's like some format of response data was changed.

**To Reproduce**
Standard run analyzer by hive or create a job on cortex.

**Expected behavior**
```
{
  "summary": {
    "taxonomies": [
      {
        "predicate": "Certificates",
        "namespace": "crt.sh",
        "value": "0",
        "level": "info"
      }
    ]
  },
  "full": {
    "certobj": {
      "domain": "ostrovok.ru",
      "result": []
    }
  },
  "success": true,
  "artifacts": [],
  "operations": []
}
```

**Complementary information**
```
{
  "errorMessage": "Error retrieving wildcard information. No JSON object could be decoded",
  "input": "{\"pap\": 1,\"tlp\": 2,\"parameters\": {},\"dataType\": \"domain\",\"data\": \"ostrovok.ru\",\"message\": \"34\",\"config\": {\"check_pap\": true,\"check_tlp\": true,\"proxy_https\": \"http://example.pl:8080\",\"proxy_http\": \"http://example.pl:8080\",\"max_tlp\": 2,\"auto_extract_artifacts\": true,\"cacerts\": null,\"jobTimeout\": 30,\"proxy\": {\"http\": \"http://example.pl:8080\",\"https\": \"http://example.pl:8080\"},\"jobCache\": 10,\"max_pap\": 2}}",
  "success": false
}
```

**Work environment**
 - Client OS: debian9
- Server OS: centos7
 - Browse type and version: n/d
- Cortex version: 3.0.0-1
 - Cortex Analyzer/Responder name:Crt_sh_Transparency_Logs_1_0
- Cortex Analyzer/Responder version:1.0

**Possible solutions**
nOpe

**Additional context**
nOpe

**Comment by nadouani** [comment]
Hello, do you have the  latest `cortexutils` version?

**Comment by KRUXLEX** [comment]
> Hello, do you have the latest `cortexutils` version?

Yup, last rebuild of docker was 2020/01/07. It works before.

**Comment by jeromeleonard** [comment]
@crackytsi and @kx499 maybe you want to have a look to adjust this analyzer to the new output of Crt.sh website.

**Comment by jeromeleonard** [comment]
@KRUXLEX the temporary fix will be added in release/2.6.0

**Comment by jeromeleonard** [comment]
Closed. @crackytsi and @kx499, please, reopen it if you plan to adjust it in regards to the new output of Crt.sh website.
